autopush/endpoint.py

```diff
@@ -219,13 +219,14 @@ def _response_err(self, fail):
         running.
 
         """
-        self.log.failure(fail, **self._client_info)
+        self.log.failure(fail, status_code=500, errno=999, **self._client_info)
         self._write_response(500, 999)
 
     def _overload_err(self, fail):
         """errBack for throughput provisioned exceptions"""
         fail.trap(ProvisionedThroughputExceededException)
-        self.log.info("Throughput Exceeded", **self._client_info)
+        self.log.info("Throughput Exceeded", status_code=503, errno=201,
+                      **self._client_info)
         self._write_response(503, 201)
 
     def _router_response(self, response):
```

Comment:
Well, _router_fail_err wasn't if exc.log_exception was set to False for whatever reason. You've got a point that if `fail` isn't a RouterException, there's a real risk that we could be burying the real problem by trying to call `log_exception` on a general exception. I'll try a few tests on that.